### PR TITLE
Fix #8070 ⁃ Probes for App menu items

### LINKED ISSUE #8070: FXIOS-1544 ⁃ Probes for App menu items
**Vision**

As a Product Manager, I want counts in place for app menu item usage, so that I can compare results after we change some of the menu items.

**Acceptance Criteria**

* tracking for clicks on Logins & Passwords
* tracking for clicks on Your Library
* tracking for clicks on Sign into Sync

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-1544)

### PR DESCRIPTION
**Pull request by dnarcese**

### CODE REVIEW

**Comment by dnarcese** [comment]
Request for data collection review form
---
1. What questions will you answer with this data?
When user clicks app menu items: Logins & Passwords, Sign into Sync, Your Library

2. Why does Mozilla need to answer these questions? Are there benefits for users? Do we need this information to address product or business requirements? Some example responses:
* Determine whether a product or platform change has an effect on user or browser behavior.

3. What alternative methods did you consider to answer these questions? Why were they not sufficient?
This is the only way to collect this data.

4. Can current instrumentation answer these questions?
No

5. List all proposed measurements and indicate the category of data collection for each measurement, using the Firefox data collection categories found on the Mozilla wiki.

Measurement Description | Data Collection Category | Tracking Bug #
-- | -- | --
user clicks Your Library | 2   |  #8070
user clicks Logins & Passwords | 2   |  #8070
user clicks Sign into Sync | 2   |  #8070

6. How long will this data be collected? Choose one of the following:
    * I want this data to be collected for 6 months initially (potentially renewable).

7. What populations will you measure?
All countries, locales, and channels that have data collection on.

8. If this data collection is default on, what is the opt-out mechanism for users?
Toggle switch for 'Send Usage Data' in the app settings.

9. Please provide a general description of how you will analyze this data.
By monitoring the percentage of users performing these actions.

10. Where do you intend to share the results of your analysis?
With the mobile team.

11. Is there a third-party tool (i.e. not Telemetry) that you are proposing to use for this data collection? If so:
No

**Comment by dnarcese** [comment]
# Data Review Form

1) Is there or will there be **documentation** that describes the schema for the ultimate data set in a public, complete, and accurate way? 

Yes, telemetry is documented in `docs/metrics.md`

2) Is there a control mechanism that allows the user to turn the data collection on and off? 

Yes, Firefox iOS provides data controls in the app settings

3) If the request is for permanent data collection, is there someone who will monitor the data over time?

This should be re-reviewed w/ Ashley Thomas every 6mo

4) Using the **[category system of data types](https://wiki.mozilla.org/Firefox/Data_Collection)** on the Mozilla wiki, what collection type of data do the requested measurements fall under?

Type 2

5) Is the data collection request for default-on or default-off?

Default on

6) Does the instrumentation include the addition of **any *new* identifiers** (whether anonymous or otherwise; e.g., username, random IDs, etc.  See the appendix for more details)?

No

7) Is the data collection covered by the existing Firefox privacy notice? 

Yes

8) Does there need to be a check-in in the future to determine whether to renew the data? (Yes/No) (If yes, set a todo reminder or file a bug if appropriate)**

Should review all probes for necessity every 6mo

9) Does the data collection use a third-party collection tool? 

No

File: Client/metrics.yaml [hunk 1]
```diff
@@ -850,3 +850,40 @@ pocket:
...
+    type: counter
+    description: |
+        Counts the number of times a user taps Your Library in the app menu
+    bugs:
+      - https://github.com/mozilla-mobile/firefox-ios/issues/1544
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-ios/pull/8167
+    notification_emails:
+      - firefox-ios@mozilla.com
+    expires: "2021-08-17"
```
Comment (nbhasin2) [inline Client/metrics.yaml:866]: Shouldn't this be 6 months from today i.e. Sepetember 17th as you want this for 6 months 
```suggestion
    expires: "2021-09-17"
```

File: Client/metrics.yaml [hunk 2]
```diff
@@ -850,3 +850,40 @@ pocket:
...
+    description: |
+        Counts the number of times a user taps Logins & Passwords
+        in the app menu
+    bugs:
+      - https://github.com/mozilla-mobile/firefox-ios/issues/1544
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-ios/pull/8167
+    notification_emails:
+      - firefox-ios@mozilla.com
+    expires: "2021-08-17"
```
Comment (nbhasin2) [inline Client/metrics.yaml:878]:
```suggestion
    expires: "2021-08-17"
```

File: Client/metrics.yaml [hunk 3]
```diff
@@ -850,3 +850,40 @@ pocket:
...
+    type: counter
+    description: |
+        Counts the number of times a user taps Sign Into Sync in the app menu
+    bugs:
+      - https://github.com/mozilla-mobile/firefox-ios/issues/1544
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-ios/pull/8167
+    notification_emails:
+      - firefox-ios@mozilla.com
+    expires: "2021-08-17"
```
Comment (nbhasin2) [inline Client/metrics.yaml:889]:
```suggestion
    expires: "2021-09-17"
```

**Review by nbhasin2 — APPROVED** [review]

File: Client/metrics.yaml (hunk 1 above)
Comment (dnarcese) [reply Client/metrics.yaml:866]: Ya I added wrong :)

**Pull request merged**